### ISSUE TITLE
Warning: Failed prop type: Invalid prop `aria-hidden` of type `boolean` supplied to `Icon`, expected `string`.

### ISSUE THREAD

**Issue by zbeedatm** [carbon-design-system/carbon#3658]
## What package(s) are you using?

- [ ] `carbon-components`
- [x] `carbon-components-react`

## Detailed description

> Describe in detail the issue you're having.
Adding a renderIcon to button, gives this warning:
Warning: Failed prop type: Invalid prop `aria-hidden` of type `boolean` supplied to `Icon`, expected `string`.

> Is this issue related to a specific component?
No

> What did you expect to happen? What happened instead? What would you like to
> see changed?
No warning!

> What browser are you working in?
Chrome

> What version of the Carbon Design System are you using?

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

## Steps to reproduce the issue

> Please create a reduced test case in CodeSandbox
> https://codesandbox.io/s/reverent-firefly-pyc03
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information

- Screenshots or code
- Notes

**Comment by zbeedatm** [comment]
Thanks for the fix. In which version it will be included?

**Comment by emyarod** [comment]
most likely v10.4.2

**Comment by HanJoHibender** [comment]
It seems this bug has reintroduced itself, I'm using @carbon/icons-react. Using the <AddAlt /> results in the following warning:

```Warning: Failed prop type: Invalid prop `aria-hidden` of type `boolean` supplied to `Icon`, expected `string`.```

I can supply more information if needed.
